cni/pkg/iptables/iptables_linux.go

```diff
@@ -77,6 +79,16 @@ func DelLoopbackRoutes(cfg *Config) error {
 	return forEachLoopbackRoute(cfg, netlink.RouteDel)
 }
 
+const ipv6DisabledLo = "/proc/sys/net/ipv6/conf/lo/disable_ipv6"
```

Comment:
Since we are running in a netns (and not a mount ns) I think this will be the (node /proc) and not the (pod /proc)?
 
 That being said, if there's a way to find the equivalent path on the host via proc using the container ID or something, we can do it that way.

Reply:
I am fairly sure this works without needing to deal with this:

```
function repro() {(
  set -x
  cat /proc/sys/net/ipv6/conf/lo/disable_ipv6
  sudo ip netns add test-v6
  sudo ip netns exec test-v6 bash -c '
    cat /proc/sys/net/ipv6/conf/lo/disable_ipv6
    sysctl -w net.ipv6.conf.all.disable_ipv6=1
    cat /proc/sys/net/ipv6/conf/lo/disable_ipv6
  '
  cat /proc/sys/net/ipv6/conf/lo/disable_ipv6
  sudo ip netns del test-v6
)}
```

```
+repro:2> cat /proc/sys/net/ipv6/conf/lo/disable_ipv6
0
+repro:3> sudo ip netns add test-v6
+repro:4> sudo ip netns exec test-v6 bash -c $'\n    cat /proc/sys/net/ipv6/conf/lo/disable_ipv6\n    sysctl -w net.ipv6.conf.all.disable_ipv6=1\n    cat /proc/sys/net/ipv6/conf/lo/disable_ipv6\n  '
0
net.ipv6.conf.all.disable_ipv6 = 1
1
+repro:9> cat /proc/sys/net/ipv6/conf/lo/disable_ipv6
0
+repro:10> sudo ip netns del test-v6

```

Reply:
Ah ok, yeah `man network_namespaces` does explicitly say `various files under /proc/sys/net` are auto-updated on netns switch. My bad.

Reply:
Yeah I thought the same, hence why I had that `repro` test ready to go :slightly_smiling_face:

Reply:
:+1: I am mostly asking the universe questions when things do not make sense to me, in order to help beat my mental model into shape resembling reality.